src/components/CheckboxOption.js

```diff
@@ -9,7 +9,7 @@ const Switch = S.default ? S.default: S // Fix: https://github.com/vitejs/vite/i
 
 class CheckboxOption extends Component {
   render() {
-    const option = this.props.options[this.props.optionKey]
+    const option = this.props.optionDefinition
```

Comment:
Can you explain the reason for "optionDefinition"? optionKey is already set on each component.

Reply:
LayerEditor creates options for shapes, layers, effects, etc (in getOptionComponent, from multiple places). The source of those options varies (it isn't always from the object that has an optionKey). I was was stuffing the optionKey with the correct value, but it felt harder than just passing the actual optionDefinition into the function.